build.sbt

```diff
@@ -27,7 +27,7 @@ lazy val buildSettings = Seq(
 lazy val rootBaseDir = ThisBuild / baseDirectory
 
 lazy val root: Project = Project(id = "root", file("."))
-  .aggregate(webapi, sipi, shared, valueObjects, userCore, userHandler, userRepo, userInterface)
+  .aggregate(webapi, sipi, shared, userCore, userHandler, userRepo, userInterface)
```

Comment:
this is missing the schema-slice and role-slice projects, so those tests are not executed on CI... could you add them?